docs/modules/publisher-directmethods.md

```diff
@@ -0,0 +1,328 @@
+[Home](../../readme.md)
+
+# Configuration via IoT Hub Direct methods
+
+OPC Publisher v2.8.2 implements the following [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application (anywhere in the world) leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks).
+
+There are some direct methods which are inherited from 2.5.x in addition to new ones.
+
+Inherited direct methods from 2.5.x:
+
+- PublishNodes_V1
+- UnpublishNodes_V1
+- UnpublishAllNodes_V1
+- GetConfiguredEndpoints_V1
+- GetConfiguredNodesOnEndpoints_V1
+- GetDiagnosticInfo_V1
+
+New direct methods:
+
+-  AddOrUpdateEndpoints_V1
```

Comment:
There are two space characters in between - and AddOrUpdateEndpoints_V1, please make it one.